README.md

```diff
@@ -76,29 +102,28 @@ You can use `BqProxyClient` as a reference, basically you just need to:
     class.
 - Register the new client in `ProxyClientFactory`.
 
-### Cloud Run deployment
-- Once your gcloud cli is configured you can deploy the application using:
-```shell
-gcloud run deploy CLOUD_RUN_SERVICE_NAME_HERE --source .
-```
-- If you enable authentication, you'll need to create a service account to use as the "invoker", the minimum required permission for that service account is "Cloud Run Invoker" for the deployed service
-- Once you configured the service account to use as the client, you can create identity tokens using:
-```shell
-gcloud auth print-identity-token [service account email address]
-```
-- The identity token is sent as the Bearer authorization header: `Authorization: Bearer IDENTITY_TOKEN`
-
-### Dev environment
+## Dev environment
 In order to test in our dev environment you need to merge your branch into `dev`, that will automatically trigger a 
 dev build and upload the image to our `pre-release-agent` repository in DockerHub.
-For now, that build is not updating the dev instance in CloudRun, in order to do that you need to run:
-```shell
-gcloud run deploy dev-apollo-agent --image montecarlodata/pre-release-agent:latest-cloudrun --region us-east1
-```
-The same build process publishes also the `generic` image with the tag `latest-generic`, so you can use it to run a
-standalone agent.
 
-### Release process
+For now, that build is not updating the dev agents for the different platforms, the easiest way to update 
+them is to connect to MC Dev environment with `dev.apollo.agent` user (credentials in 1Pwd), go to
+Settings -> Integrations -> Agents & Data Store and update the (or use the `agents upgrade` command in MC CLI).
+
```

Comment:
Similar about these comments, we might need to move them elsewhere?

Reply:
You can just link to our public docs for the process
- AWS: https://docs.getmontecarlo.com/docs/create-and-register-an-aws-agent#how-do-i-upgrade-the-agent
- GCP: https://docs.getmontecarlo.com/docs/create-and-register-a-gcp-agent#how-do-i-upgrade-the-agent